server/config.go

```diff
@@ -102,7 +102,9 @@ type Config struct {
 	StartLoadtestTag     string
 	StartLoadtestMessage string
 
-	CLAExclusionsList     []string
+	CLAExclusionsList      []string
+	CLAGithubStatusContext string
```

Comment:
0/5 if this needs to be configurable. Maybe using a static string is fine

Reply:
I have already contexts in the config. This makes it consistent.